config/schema/base_document_type.json

```diff
@@ -10,13 +10,16 @@
     "latest_change_note",
     "link",
     "mainstream_browse_pages",
+    "mainstream_browse_page_content_ids",
     "organisations",
+    "organisation_content_ids",
     "policies",
     "popularity",
     "public_timestamp",
     "publishing_app",
     "rendering_app",
     "specialist_sectors",
+    "topics",
```

Comment:
That was actually its original name and then I changed it to `topics`. I agree it would be clearer here, but I thought that as nothing else is called `topics` it would be 1 less thing to worry about later. But I'm definitely happy to change it again to look like the other fields.